src/nodedef.h

```diff
@@ -484,3 +488,5 @@ class NodeResolver {
 };
 
 #endif
+
+// kate: space-indent off; indent-mode cstyle;
```

Comment:
@nerzhul no modelines? Okay, will remove. (later as my editor became crazy recently, it refuses to insert tabs in `.h` files otherwise).